app/_locales/en/messages.json

```diff
@@ -3227,6 +3227,10 @@
     "message": "Added on $1 from $2",
     "description": "$1 represents the date the snap was installed, $2 represents which origin installed the snap."
   },
+  "snapContent": {
+    "message": "This content is coming from $1",
+    "description": "This is shown when a snap shows tx insight information in the confirmation UI. $1 is a link to the snap's settings page with the link text being the name of the snap."
```

Comment:
Just double checking that we want to use the abbreviation of `tx` as `transaction` cc. @coreyjanssen